metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/PluginStatus.java

```diff
@@ -3,21 +3,25 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 
 /**
+ * The status that a plugin can have.
+ *
  * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
  * @since 2017-05-26
  */
 public enum PluginStatus {
-  INQUEUE, CLEANING, RUNNING, FINISHED, CANCELLED, FAILED;
+  INQUEUE, CLEANING, RUNNING, FINISHED, CANCELLED, FAILED, PENDING;
```

Comment:
I think there are some places where you should treat PENDING the same as RUNNING. I found:
1. WorkflowExecution.setAllRunningAndInqueuePluginsToCancelled()
2. WorkflowExecution.checkAndSetAllRunningAndInqueuePluginsToCancelledIfOnePluginHasFailed()
3. WorkflowExecutor.runInqueueOrRunningStateWorkflowExecution()

Reply:
Indeed, I've fixed that.